src/Options.cc

```diff
@@ -124,9 +124,15 @@ void usage(const char* prog) {
     printf(
         "    -0|--optimize-files=<pat>       | enable script optimization for all "
         "functions in files with names containing the given pattern\n");
+    printf(
+        "       --no-opt-files=<pat>         | skip script optimization for all "
```

Comment:
I think these both should be `-no-optimize-*` to match the existing options.

Reply:
Wordy to type, but I do appreciate the point. Changed in pending push